ui/i18n.js

```diff
@@ -0,0 +1,31 @@
+
+// cross-browser connection to extension i18n API
+
+const chrome = chrome || null
+const browser = browser || null
+let getMessage = null
+
+if ((chrome && chrome.i18n && chrome.i18n.getMessage) ||
+    (browser && browser.i18n && browser.i18n.getMessage)) {
+  getMessage = (chrome || browser).i18n.getMessage
+} else {
+  // fallback function
+  console.warn('browser.i18n API not available?')
```

Comment:
We use a global `loglevel` instance. We should document it on our contributing guide. Correct form here will be `log.warn(message)`.